features/search_functionality.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Search Functionality
+  As a user,
+  In order to filter the images
+  I should be able to search
+
+Background:
+  Given that i am on the landing page
+  And there is a field "search_field"
+  Given The following photos exist
+    | title   | price   | image_path                              |
+    | Sporty1 | 30      | http://lorempixel.com/400/200/sports/1/ |
+    | Sporty2 | 40      | http://lorempixel.com/400/200/sports/2/ |
+    | Sporty3 | 50      | http://lorempixel.com/400/200/sports/3/ |
+    | Sporty4 | 60      | http://lorempixel.com/400/200/sports/4/ |
+
+Scenario: Search from the landing page [Happy path]
+  When I fill in "search_field" with "Sporty1"
```

Comment:
Can't we use `When I fill in "Search" with "Sporty1"`
You want to write a field name that the user will actually see when they visit the page